modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -387,10 +473,12 @@ PERF_TEST_P_(DivPerfTest, TestPerformance)
 
 PERF_TEST_P_(DivCPerfTest, TestPerformance)
 {
-    Size sz = get<0>(GetParam());
-    MatType type = get<1>(GetParam());
-    int dtype = get<2>(GetParam());
-    cv::GCompileArgs compile_args = get<3>(GetParam());
+    compare_f cmpF;
+    cv::Size sz;
+    MatType type = -1;
+    int dtype = -1;
+    cv::GCompileArgs compile_args;
+    std::tie(cmpF, sz, type, dtype, compile_args) = GetParam();
```

Comment:
Please create `double scale` variable and add its initialization via test configuration parameter. Example is `DivPerfTest` above.

Reply:
> add its initialization via test configuration parameter.

This comment is still not applied.

Reply:
Applied